File: auto-docs/executables/python/basic_error_bar.py
import plotly.plotly as py
from plotly.graph_objs import *
py.sign_in('TestBot', 'r1neazxo9w')

data = Data([
    Scatter(
        x=[0, 1, 2],
        y=[6, 10, 2],
        error_y=ErrorY(
            type='data',
            array=[1, 2, 3],
            visible=True
        )
    )
])
plot_url = py.plot(data, filename='basic-error-bar', auto_open=False)

File: auto-docs/executables/nodejs/basic_error_bar.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var data = [
  {
    x: [0, 1, 2], 
    y: [6, 10, 2], 
    error_y: {
      type: "data", 
      array: [1, 2, 3], 
      visible: true
    }, 
    type: "scatter"
  }
];

var graph_options = {filename: "basic-error-bar", fileopt: "overwrite", auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});